WooCommerce/Classes/ViewRelated/Dashboard/Settings/Plugins/PluginListViewController.swift

```diff
@@ -43,7 +43,7 @@ final class PluginListViewController: UIViewController {
         configureNavigation()
         configureTableView()
         configureGhostTableView()
-        configureViewModel()
+        syncPlugins(onCompletion: configureViewModel)
```

Comment:
Could you explain a little bit why do we need to configure the view model after plugins are synced? 

The `VM` is in charge of fetching the plugins, but this call stack implies that something else syncs the plugin and then we configure the view model.

I think we should aim to sync the plugins inside the `configureViewModel` method.

Reply:
Something like
```swift
func configureViewModel() {
    viewModel.observePlugins { [weak self] in
        self?.tableView.reloadData()
    }

    syncPlugins(onCompletion: nil)
}
```

Reply:
The only reason why I did it in that way is to first sync the data and second initialize the resultController. But in the end the result is exactly the same. I'm going to change it.